public/org

```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet" href="/css/org.css">
 
   <title>Org</title>
+  <link rel="icon" href="article-24px.svg" type="image/svg+xml">
```

Comment:
You don't need this change remember.

So from next time, always make a new branch from `master`
Right now I am merging, but always work for other issues on other branches